js/script.js

```diff
@@ -1144,7 +1144,39 @@ var validCombos = {
     "Follow build instructions from <a href='https://aka.ms/build-ort-snpe' target='_blank'>here</a>",
 
     "android,C++,ARM64,SNPE":
-    "Follow build instructions from <a href='https://aka.ms/build-ort-snpe' target='_blank'>here</a>"
+        "Follow build instructions from <a href='https://aka.ms/build-ort-snpe' target='_blank'>here</a>",
+
+    //Xnnpack
+    "android,C-API,ARM64,XNNPACK":
+        "Follow build instructions from <a href='https://aka.ms/build-ort-xnnpack' target='_blank'>here</a>",
+
+    "android,C++,ARM64,XNNPACK":
+        "Follow build instructions from <a href='https://aka.ms/build-ort-xnnpack' target='_blank'>here</a>",
+
+    "android,Java,ARM64,XNNPACK":
+        "Add a dependency on <a href='https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-android' target='_blank'>com.microsoft.onnxruntime:onnxruntime-android</a> or <a href='https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-mobile' target='_blank'>com.microsoft.onnxruntime:onnxruntime-mobile</a> using Maven/Gradle and refer to the <a href='https://onnxruntime.ai/docs/tutorials/mobile/' target='_blank'>mobile deployment guide</a>",
+
```

Comment:
IIRC we only include xnnpack in the full package.

Reply:
@wejoncy please address this comment

Reply:
Hi @faxu  I talked to Scott already, it's some troubles to build XNNPACK for IOS platform. So, it may be acceptable to update it in a new PR as well.

Or I can tackle the build issue first then update IOS related package along with this PR if you want.